### PR TITLE
feat(output): update the HTML output

### PR DESCRIPTION
**Pull request by ekifel**
Issue #548 

Updated: 
- HTML output to be consistent with CLI output
- Warnings and `--help` descriptions

Added: 
- Support dynamic columns

### CODE REVIEW

File: internal/output/templates.go [hunk 1]
```diff
@@ -157,14 +166,76 @@ iVBORw0KGgoAAAANSUhEUgAAAMAAAADACAMAAABlApw1AAAABGdBTUEAALGPC/xhBQAAAAFzUkdCAK7O
...
+        <td class="price">{{.CostComponent.Price | formatPrice }}</td>
+      {{end}}
+      {{if contains .Fields "hourlyCost"}}
+        <td class="hourly-cost">{{.CostComponent.HourlyCost | formatCost2DP}}</td>
+      {{end}}
+      {{if contains .Fields "monthlyCost"}}
+        <td class="monthly-cost">{{.CostComponent.MonthlyCost | formatCost2DP}}</td>
+      {{end}}
+    {{else}}
+      <td colspan="{{len .Fields}}" class="usage-cost">Cost depends on usage: {{.CostComponent.Price | formatPrice}} per {{.CostComponent.Unit}}</td>
```
Comment (aliscott) [inline internal/output/templates.go:186]: Should we make the text color `#6b7280` here so it's a little faded like in the CLI?

File: internal/output/templates.go [hunk 2]
```diff
@@ -157,14 +166,76 @@ iVBORw0KGgoAAAANSUhEUgAAAMAAAADACAMAAABlApw1AAAABGdBTUEAALGPC/xhBQAAAAFzUkdCAK7O
...
+  {{if contains .Fields "monthlyCost"}}
+    <td class="monthly-cost">Monthly Cost</td>
+  {{end}}
+{{end}}
+
+{{define "projectBlock"}}
+  {{$fields := .Options.Fields}}
+  <p class="project-name">Project: {{.Project.Path}}</p>
+  <div class="table-border">
+    <table>
```
Comment (aliscott) [inline internal/output/templates.go:214]: Maybe we should add a sensible set width to the table so the tables are the same width.

**Review by aliscott — COMMENTED** [review]
This looks awesome! Thanks @ekifel.

File: internal/output/templates.go [hunk 3]
```diff
@@ -38,8 +38,8 @@ a {
 }
 
 table {
+  border: 1px solid gray;
```
[the remark this answers is not in the document]
Comment (ekifel) [reply internal/output/templates.go:41]: Because I changed it while trying to figure out with one problem and forgot about it. I'll bring it back

**Comment by aliscott** [comment]
Thanks @ekifel! 😄  Merging this into the v0.9 branch which we will use going forward for v0.9 features.

**Pull request merged**